module,execution_time (sec),success,timestamp,error,error_short
cowidev.vax.batch.argentina,15.99,True,2023-04-03T07:00:23,,
cowidev.vax.batch.australia,0.23,False,2023-04-03T07:01:50,"Traceback (most recent call last):
  File ""/home/owid/covid-19-data/scripts/src/cowidev/cmd/commons/get.py"", line 47, in run
    module.main()
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/batch/australia.py"", line 244, in main
    Australia().export()
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/batch/australia.py"", line 216, in export
    df = self.read().pipe(self.pipeline)  # Use with attach=False
  File ""/home/owid/covid-19-data/scripts/venv/lib/python3.10/site-packages/pandas/core/generic.py"", line 5430, in pipe
    return com.pipe(self, func, *args, **kwargs)
  File ""/home/owid/covid-19-data/scripts/venv/lib/python3.10/site-packages/pandas/core/common.py"", line 471, in pipe
    return func(obj, *args, **kwargs)
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/batch/australia.py"", line 160, in pipeline
    .pipe(self.make_monotonic)
  File ""/home/owid/covid-19-data/scripts/venv/lib/python3.10/site-packages/pandas/core/generic.py"", line 5430, in pipe
    return com.pipe(self, func, *args, **kwargs)
  File ""/home/owid/covid-19-data/scripts/venv/lib/python3.10/site-packages/pandas/core/common.py"", line 471, in pipe
    return func(obj, *args, **kwargs)
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/utils/base.py"", line 120, in make_monotonic
    return mkm(
  File ""/home/owid/covid-19-data/scripts/src/cowidev/utils/utils.py"", line 23, in make_monotonic
    return make_monotonic_new(df, column_date, column_metrics, max_removed_rows)
  File ""/home/owid/covid-19-data/scripts/src/cowidev/utils/utils.py"", line 94, in make_monotonic_new
    raise Exception(
Exception: 1 chunks of lengths 19 have been NaNed for metric people_vaccinated. That is more than maximum allowed (10) by make_monotonic() - check the data. Check dates ['2023-03-16']
",1 chunks of lengths 19 have been NaNed for metric people_vaccinated. That is more than maximum allowed (10) by make_monotonic() - check the data. Check dates ['2023-03-16']
cowidev.vax.batch.austria_old,,,2023-04-03T07:00:07,,
cowidev.vax.batch.belgium,8.47,True,2023-04-03T07:00:15,,
cowidev.vax.batch.bolivia,4.5,True,2023-04-03T07:00:11,,
cowidev.vax.batch.canada,5.14,False,2023-04-03T07:01:55,"Traceback (most recent call last):
  File ""/home/owid/covid-19-data/scripts/src/cowidev/cmd/commons/get.py"", line 47, in run
    module.main()
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/batch/canada.py"", line 282, in main
    Canada().export()
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/batch/canada.py"", line 270, in export
    df = df.pipe(self.pipeline, df_man)
  File ""/home/owid/covid-19-data/scripts/venv/lib/python3.10/site-packages/pandas/core/generic.py"", line 5430, in pipe
    return com.pipe(self, func, *args, **kwargs)
  File ""/home/owid/covid-19-data/scripts/venv/lib/python3.10/site-packages/pandas/core/common.py"", line 471, in pipe
    return func(obj, *args, **kwargs)
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/batch/canada.py"", line 250, in pipeline
    .pipe(self.pipe_make_monotonic)[
  File ""/home/owid/covid-19-data/scripts/venv/lib/python3.10/site-packages/pandas/core/generic.py"", line 5430, in pipe
    return com.pipe(self, func, *args, **kwargs)
  File ""/home/owid/covid-19-data/scripts/venv/lib/python3.10/site-packages/pandas/core/common.py"", line 471, in pipe
    return func(obj, *args, **kwargs)
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/batch/canada.py"", line 230, in pipe_make_monotonic
    df = df.pipe(self.make_monotonic, max_removed_rows=self.max_removed_rows)
  File ""/home/owid/covid-19-data/scripts/venv/lib/python3.10/site-packages/pandas/core/generic.py"", line 5430, in pipe
    return com.pipe(self, func, *args, **kwargs)
  File ""/home/owid/covid-19-data/scripts/venv/lib/python3.10/site-packages/pandas/core/common.py"", line 471, in pipe
    return func(obj, *args, **kwargs)
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/utils/base.py"", line 120, in make_monotonic
    return mkm(
  File ""/home/owid/covid-19-data/scripts/src/cowidev/utils/utils.py"", line 23, in make_monotonic
    return make_monotonic_new(df, column_date, column_metrics, max_removed_rows)
  File ""/home/owid/covid-19-data/scripts/src/cowidev/utils/utils.py"", line 94, in make_monotonic_new
    raise Exception(
Exception: 1 chunks of lengths 36 have been NaNed for metric people_vaccinated. That is more than maximum allowed (22) by make_monotonic() - check the data. Check dates ['2023-01-26']
",1 chunks of lengths 36 have been NaNed for metric people_vaccinated. That is more than maximum allowed (22) by make_monotonic() - check the data. Check dates ['2023-01-26']
cowidev.vax.batch.chile,1.12,True,2023-04-03T07:00:08,,
cowidev.vax.batch.czechia,33.99,True,2023-04-03T07:00:41,,
cowidev.vax.batch.denmark,2.56,False,2023-04-03T08:31:16,"Traceback (most recent call last):
  File ""/home/lucas/repos/covid-19-data/scripts/src/cowidev/cmd/commons/get.py"", line 47, in run
    module.main()
  File ""/home/lucas/repos/covid-19-data/scripts/src/cowidev/vax/batch/denmark.py"", line 246, in main
    Denmark().export()
  File ""/home/lucas/repos/covid-19-data/scripts/src/cowidev/vax/batch/denmark.py"", line 227, in export
    df = self.read(index).pipe(self.pipeline, df_current).pipe(self.make_monotonic)
  File ""/home/lucas/repos/covid-19-data/scripts/src/cowidev/vax/batch/denmark.py"", line 43, in read
    url = self._parse_link_zip()
  File ""/home/lucas/repos/covid-19-data/scripts/src/cowidev/vax/batch/denmark.py"", line 68, in _parse_link_zip
    url = soup.find(""a"", string=""Download her"").get(""href"")
AttributeError: 'NoneType' object has no attribute 'get'
",'NoneType' object has no attribute 'get'
cowidev.vax.batch.ecdc,35.62,True,2023-04-03T08:36:06,,
cowidev.vax.batch.ecuador,,,2023-04-03T07:00:07,,
cowidev.vax.batch.estonia,0.57,True,2023-04-03T07:00:07,,
cowidev.vax.batch.france,8.85,True,2023-04-03T07:00:16,,
cowidev.vax.batch.germany,0.15,True,2023-04-03T07:00:07,,
cowidev.vax.batch.greece,12.37,True,2023-04-03T07:00:19,,
cowidev.vax.batch.hong_kong,14.79,True,2023-04-03T07:00:22,,
cowidev.vax.batch.indonesia,,,2023-04-03T07:00:07,,
cowidev.vax.batch.ireland,4.18,True,2023-04-03T07:00:11,,
cowidev.vax.batch.israel,9.67,True,2023-04-03T07:00:17,,
cowidev.vax.batch.italy,9.63,True,2023-04-03T07:00:17,,
cowidev.vax.batch.japan,36.01,True,2023-04-03T07:00:43,,
cowidev.vax.batch.jersey,4.78,True,2023-04-03T07:00:12,,
cowidev.vax.batch.latvia,66.1,True,2023-04-03T07:01:13,,
cowidev.vax.batch.lithuania,5.35,True,2023-04-03T07:00:12,,
cowidev.vax.batch.luxembourg,2.25,True,2023-04-03T07:00:09,,
cowidev.vax.batch.malaysia,0.99,True,2023-04-03T07:00:08,,
cowidev.vax.batch.malta,1.22,True,2023-04-03T07:00:08,,
cowidev.vax.batch.netherlands,,,2023-04-03T07:00:07,,
cowidev.vax.batch.new_zealand,15.02,True,2023-04-03T07:00:22,,
cowidev.vax.batch.norway,0.34,True,2023-04-03T07:00:07,,
cowidev.vax.batch.peru,4.98,True,2023-04-03T07:00:12,,
cowidev.vax.batch.portugal,,,2023-04-03T07:00:07,,
cowidev.vax.batch.romania,33.56,True,2023-04-03T07:00:40,,
cowidev.vax.batch.saudi_arabia,9.72,True,2023-04-03T07:00:17,,
cowidev.vax.batch.singapore,5.09,True,2023-04-03T07:00:12,,
cowidev.vax.batch.slovakia,49.78,True,2023-04-03T07:00:57,,
cowidev.vax.batch.slovenia,3.94,True,2023-04-03T07:00:11,,
cowidev.vax.batch.south_korea,70.73,True,2023-04-03T07:01:18,,
cowidev.vax.batch.spc,21.02,True,2023-04-03T07:00:28,,
cowidev.vax.batch.sweden,2.59,True,2023-04-03T20:52:19,,
cowidev.vax.batch.switzerland,84.94,True,2023-04-03T07:01:33,,
cowidev.vax.batch.trinidad_and_tobago,5.51,True,2023-04-03T07:00:14,,
cowidev.vax.batch.ukraine,,,2023-04-03T07:00:09,,
cowidev.vax.batch.united_kingdom,13.97,True,2023-04-03T07:00:23,,
cowidev.vax.batch.united_states,9.02,False,2023-04-03T08:37:04,"Traceback (most recent call last):
  File ""/home/lucas/repos/covid-19-data/scripts/src/cowidev/cmd/commons/get.py"", line 47, in run
    module.main()
  File ""/home/lucas/repos/covid-19-data/scripts/src/cowidev/vax/batch/united_states.py"", line 192, in main
    UnitedStates().export()
  File ""/home/lucas/repos/covid-19-data/scripts/src/cowidev/vax/batch/united_states.py"", line 179, in export
    df_manufacturer = self.read_manufacturer().pipe(self.pipeline_manufacturer)
  File ""/home/lucas/.pyenv/versions/3.10.9/lib/python3.10/site-packages/pandas/core/generic.py"", line 5430, in pipe
    return com.pipe(self, func, *args, **kwargs)
  File ""/home/lucas/.pyenv/versions/3.10.9/lib/python3.10/site-packages/pandas/core/common.py"", line 471, in pipe
    return func(obj, *args, **kwargs)
  File ""/home/lucas/repos/covid-19-data/scripts/src/cowidev/vax/batch/united_states.py"", line 172, in pipeline_manufacturer
    df = df.pipe(self.make_monotonic, ""vaccine"")
  File ""/home/lucas/.pyenv/versions/3.10.9/lib/python3.10/site-packages/pandas/core/generic.py"", line 5430, in pipe
    return com.pipe(self, func, *args, **kwargs)
  File ""/home/lucas/.pyenv/versions/3.10.9/lib/python3.10/site-packages/pandas/core/common.py"", line 471, in pipe
    return func(obj, *args, **kwargs)
  File ""/home/lucas/repos/covid-19-data/scripts/src/cowidev/vax/utils/base.py"", line 109, in make_monotonic
    _df = mkm(
  File ""/home/lucas/repos/covid-19-data/scripts/src/cowidev/utils/utils.py"", line 23, in make_monotonic
    return make_monotonic_new(df, column_date, column_metrics, max_removed_rows)
  File ""/home/lucas/repos/covid-19-data/scripts/src/cowidev/utils/utils.py"", line 94, in make_monotonic_new
    raise Exception(
Exception: 1 chunks of lengths 26 have been NaNed for metric total_vaccinations. That is more than maximum allowed (10) by make_monotonic() - check the data. Check dates ['2023-03-22']
",1 chunks of lengths 26 have been NaNed for metric total_vaccinations. That is more than maximum allowed (10) by make_monotonic() - check the data. Check dates ['2023-03-22']
cowidev.vax.batch.uruguay,7.51,True,2023-04-03T07:00:17,,
cowidev.vax.batch.zimbabwe,0.28,True,2023-04-03T07:02:04,,
cowidev.vax.incremental.africacdc,,,2023-04-03T07:00:11,,
cowidev.vax.incremental.antigua_barbuda,,,2023-04-03T07:00:11,,
cowidev.vax.incremental.aruba,1.91,True,2023-04-03T07:00:13,,
cowidev.vax.incremental.austria,,,2023-04-03T07:00:11,,
cowidev.vax.incremental.azerbaijan,,,2023-04-03T07:00:11,,
cowidev.vax.incremental.bahrain,,,2023-04-03T07:00:11,,
cowidev.vax.incremental.bangladesh,41.24,True,2023-04-03T07:00:52,,
cowidev.vax.incremental.barbados,,,2023-04-03T07:00:11,,
cowidev.vax.incremental.brazil,1.21,True,2023-04-03T07:00:13,,
cowidev.vax.incremental.bulgaria,7.26,True,2023-04-03T07:00:19,,
cowidev.vax.incremental.china,18.59,True,2023-04-03T07:00:30,,
cowidev.vax.incremental.costa_rica,,,2023-04-03T07:00:12,,
cowidev.vax.incremental.croatia,2.12,True,2023-04-03T07:00:14,,
cowidev.vax.incremental.cuba,5.38,True,2023-04-03T07:00:18,,
cowidev.vax.incremental.curacao,2.19,True,2023-04-03T07:00:15,,
cowidev.vax.incremental.cyprus,3.24,True,2023-04-03T07:00:16,,
cowidev.vax.incremental.dominican_republic,11.3,False,2023-04-03T07:02:15,"Traceback (most recent call last):
  File ""/home/owid/covid-19-data/scripts/src/cowidev/cmd/commons/get.py"", line 47, in run
    module.main()
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/incremental/dominican_republic.py"", line 113, in main
    DominicanRepublic().export()
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/incremental/dominican_republic.py"", line 68, in export
    data = self.read().pipe(self.pipeline)
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/incremental/dominican_republic.py"", line 26, in read
    metrics = self._parse_metrics(driver)
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/incremental/dominican_republic.py"", line 35, in _parse_metrics
    metric_candidates = _find_potential_metrics(driver)
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/incremental/dominican_republic.py"", line 92, in _find_potential_metrics
    h3 = _find_h3(driver)
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/incremental/dominican_republic.py"", line 82, in _find_h3
    raise ValueError(""More than one element found!"")
ValueError: More than one element found!
",More than one element found!
cowidev.vax.incremental.el_salvador,6.27,True,2023-04-03T07:00:20,,
cowidev.vax.incremental.faeroe_islands,1.34,True,2023-04-03T07:00:16,,
cowidev.vax.incremental.fiji,0.0,True,2023-04-03T07:00:15,,
cowidev.vax.incremental.finland,0.83,True,2023-04-03T07:00:16,,
cowidev.vax.incremental.gabon,,,2023-04-03T07:00:16,,
cowidev.vax.incremental.georgia,1.97,True,2023-04-03T07:00:18,,
cowidev.vax.incremental.greenland,0.63,False,2023-04-03T07:02:15,"Traceback (most recent call last):
  File ""/home/owid/covid-19-data/scripts/src/cowidev/cmd/commons/get.py"", line 47, in run
    module.main()
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/incremental/greenland.py"", line 76, in main
    Greenland().export()
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/incremental/greenland.py"", line 63, in export
    data = self.read().pipe(self.pipeline)
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/incremental/greenland.py"", line 17, in read
    data = self._parse_data(soup)
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/incremental/greenland.py"", line 21, in _parse_data
    return {**self._parse_data_metrics(soup), **self._parse_data_date(soup)}
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/incremental/greenland.py"", line 31, in _parse_data_metrics
    assert len(counter_1) == len(counter_2) == 1
AssertionError
",
cowidev.vax.incremental.guatemala,,,2023-04-03T07:00:16,,
cowidev.vax.incremental.guernsey,5.99,True,2023-04-03T07:00:22,,
cowidev.vax.incremental.hungary,79.08,True,2023-04-03T07:01:35,,
cowidev.vax.incremental.iceland,1.12,True,2023-04-03T07:00:18,,
cowidev.vax.incremental.india,4.61,True,2023-04-03T07:00:21,,
cowidev.vax.incremental.iran,,,2023-04-03T07:00:17,,
cowidev.vax.incremental.isle_of_man,3.01,True,2023-04-03T07:00:20,,
cowidev.vax.incremental.jamaica,7.94,True,2023-04-03T07:00:25,,
cowidev.vax.incremental.kazakhstan,6.79,True,2023-04-03T07:00:24,,
cowidev.vax.incremental.kosovo,0.12,False,2023-04-03T07:02:16,"Traceback (most recent call last):
  File ""/home/owid/covid-19-data/scripts/src/cowidev/cmd/commons/get.py"", line 47, in run
    module.main()
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/incremental/kosovo.py"", line 82, in main
    Kosovo().export()
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/incremental/kosovo.py"", line 77, in export
    df = self.read().pipe(self.pipeline)
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/incremental/kosovo.py"", line 22, in read
    soup = get_soup(self.source_url, verify=False)
  File ""/home/owid/covid-19-data/scripts/src/cowidev/utils/web/scraping.py"", line 83, in get_soup
    response = get_response(source, request_method, use_proxy, **kwargs)
  File ""/home/owid/covid-19-data/scripts/src/cowidev/utils/web/scraping.py"", line 52, in get_response
    raise ValueError(
ValueError: Source https://msh.rks-gov.net/sq/statistikat-covid-19/ not reached! Error code 404 Not Found
",Source https://msh.rks-gov.net/sq/statistikat-covid-19/ not reached! Error code 404 Not Found
cowidev.vax.incremental.kyrgyzstan,5.76,True,2023-04-03T07:00:23,,
cowidev.vax.incremental.laos,6.13,True,2023-04-03T07:00:24,,
cowidev.vax.incremental.macao,13.27,False,2023-04-03T07:02:29,"Traceback (most recent call last):
  File ""/home/owid/covid-19-data/scripts/src/cowidev/cmd/commons/get.py"", line 47, in run
    module.main()
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/incremental/macao.py"", line 89, in main
    Macao().export()
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/incremental/macao.py"", line 74, in export
    data = self.read()
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/incremental/macao.py"", line 25, in read
    data = self._parse_data(iframe_url, driver)
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/incremental/macao.py"", line 37, in _parse_data
    elem = self._get_elem_div(driver, ""Total doses administered (local and non-local)"")
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/incremental/macao.py"", line 65, in _get_elem_div
    assert len(elem) == 1
AssertionError
",
cowidev.vax.incremental.moldova,,,2023-04-03T07:00:19,,
cowidev.vax.incremental.monaco,83.64,True,2023-04-03T07:01:43,,
cowidev.vax.incremental.mongolia,1.29,True,2023-04-03T07:00:21,,
cowidev.vax.incremental.montenegro,0.96,True,2023-04-03T07:00:21,,
cowidev.vax.incremental.morocco,,,2023-04-03T07:00:20,,
cowidev.vax.incremental.myanmar,,,2023-04-03T07:00:20,,
cowidev.vax.incremental.nepal,,,2023-04-03T07:00:20,,
cowidev.vax.incremental.north_macedonia,,,2023-04-03T07:00:20,,
cowidev.vax.incremental.northern_cyprus,4.56,True,2023-04-03T07:00:25,,
cowidev.vax.incremental.paho,82.42,False,2023-04-03T07:03:51,"Traceback (most recent call last):
  File ""/home/owid/covid-19-data/scripts/src/cowidev/cmd/commons/get.py"", line 47, in run
    module.main()
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/incremental/paho.py"", line 195, in main
    PAHO().export()
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/incremental/paho.py"", line 190, in export
    df = self.read().pipe(self.pipeline)
  File ""/home/owid/covid-19-data/scripts/venv/lib/python3.10/site-packages/pandas/core/generic.py"", line 5430, in pipe
    return com.pipe(self, func, *args, **kwargs)
  File ""/home/owid/covid-19-data/scripts/venv/lib/python3.10/site-packages/pandas/core/common.py"", line 471, in pipe
    return func(obj, *args, **kwargs)
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/incremental/paho.py"", line 167, in pipeline
    df.pipe(self.pipe_check_columns)
  File ""/home/owid/covid-19-data/scripts/venv/lib/python3.10/site-packages/pandas/core/generic.py"", line 5430, in pipe
    return com.pipe(self, func, *args, **kwargs)
  File ""/home/owid/covid-19-data/scripts/venv/lib/python3.10/site-packages/pandas/core/common.py"", line 471, in pipe
    return func(obj, *args, **kwargs)
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/incremental/paho.py"", line 122, in pipe_check_columns
    raise ValueError(f""Missing column fields: {columns_missing}. Present columns are: {df.columns}"")
ValueError: Missing column fields: {'1st additional dose', 'Single dose', 'Country/ Territory', '2nd additional dose', 'Total doses', 'Country code', 'Second dose', 'First dose'}. Present columns are: Index(['date,location,total_vaccinations,people_vaccinated,people_fully_vaccinated,total_boosters,new_vaccinations,new_vaccinations_interpolated,new_people_vaccinated_interpolated,new_vaccinations_smoothed,new_people_vaccinated_smoothed', 'date'], dtype='object')
","Missing column fields: {'1st additional dose', 'Single dose', 'Country/ Territory', '2nd additional dose', 'Total doses', 'Country code', 'Second dose', 'First dose'}. Present columns are: Index(['date,location,total_vaccinations,people_vaccinated,people_fully_vaccinated,total_boosters,new_vaccinations,new_vaccinations_interpolated,new_people_vaccinated_interpolated,new_vaccinations_smoothed,new_people_vaccinated_smoothed', 'date'], dtype='object')"
cowidev.vax.incremental.pakistan,,,2023-04-03T07:00:21,,
cowidev.vax.incremental.philippines,,,2023-04-03T07:00:21,,
cowidev.vax.incremental.poland,1.57,True,2023-04-03T07:00:22,,
cowidev.vax.incremental.qatar,,,2023-04-03T07:00:21,,
cowidev.vax.incremental.russia,18.36,True,2023-04-03T07:00:40,,
cowidev.vax.incremental.saint_lucia,,,2023-04-03T07:00:22,,
cowidev.vax.incremental.serbia,9.77,True,2023-04-03T07:00:31,,
cowidev.vax.incremental.south_africa,,,2023-04-03T07:00:22,,
cowidev.vax.incremental.spain,7.38,True,2023-04-03T07:00:29,,
cowidev.vax.incremental.sri_lanka,,,2023-04-03T07:00:22,,
cowidev.vax.incremental.suriname,14.84,True,2023-04-03T07:00:37,,
cowidev.vax.incremental.taiwan,8.46,False,2023-04-03T07:04:00,"Traceback (most recent call last):
  File ""/home/owid/covid-19-data/scripts/src/cowidev/cmd/commons/get.py"", line 47, in run
    module.main()
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/incremental/taiwan.py"", line 179, in main
    Taiwan().export()
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/incremental/taiwan.py"", line 165, in export
    data = self.read().pipe(self.pipeline)
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/incremental/taiwan.py"", line 35, in read
    df = self._parse_table(url_pdf)
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/incremental/taiwan.py"", line 62, in _parse_table
    raise ValueError(f""Table 1: format has changed! It has shape {df.shape} instead of {shape_expected}"")
ValueError: Table 1: format has changed! It has shape (42, 4) instead of (39, 4)
","Table 1: format has changed! It has shape (42, 4) instead of (39, 4)"
cowidev.vax.incremental.thailand,6.93,True,2023-04-03T07:00:29,,
cowidev.vax.incremental.turkey,11.2,False,2023-04-03T07:04:11,"Traceback (most recent call last):
  File ""/home/owid/covid-19-data/scripts/src/cowidev/cmd/commons/get.py"", line 47, in run
    module.main()
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/incremental/turkey.py"", line 60, in main
    data = read(source).pipe(pipeline)
  File ""/home/owid/covid-19-data/scripts/src/cowidev/vax/incremental/turkey.py"", line 19, in read
    soup = get_soup(source)
  File ""/home/owid/covid-19-data/scripts/src/cowidev/utils/web/scraping.py"", line 83, in get_soup
    response = get_response(source, request_method, use_proxy, **kwargs)
  File ""/home/owid/covid-19-data/scripts/src/cowidev/utils/web/scraping.py"", line 52, in get_response
    raise ValueError(
ValueError: Source https://covid19asi.saglik.gov.tr/ not reached! Error code 504 Gateway Timeout
",Source https://covid19asi.saglik.gov.tr/ not reached! Error code 504 Gateway Timeout
cowidev.vax.incremental.united_arab_emirates,,,2023-04-03T07:00:23,,
cowidev.vax.incremental.vietnam,,,2023-04-03T07:00:23,,
cowidev.vax.incremental.who,34.84,True,2023-04-03T07:00:58,,
cowidev.vax.incremental.zambia,1.09,True,2023-04-03T07:00:25,,
